components/provisioner/e2e_test/test.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+echo "Entering dir: $SCRIPT_DIR"
+cd $SCRIPT_DIR
+
+service docker start
```

Comment:
When running this command I see in the output 

```
Entering dir: (...) control-plane/components/provisioner/e2e_test
./test.sh: line 7: service: command not found
```
As I understand it's not the way how you start docker on MacOs but this line is need for the CI/CD.
Just wanted to bring it to your attention but I believe that it can really stay like it is as the error message is clear.

Reply:
fixed